Source/JavaScriptCore/Configurations/JavaScriptCore.xcconfig

```diff
@@ -24,6 +24,11 @@
 #include "BaseTarget.xcconfig"
 
 DEFINES_MODULE = YES;
+ENABLE_MODULE_VERIFIER = $(USE_INTERNAL_SDK);
```

Comment:
I think you can just make this YES. Where it's not supported in the public SDK, the setting is ignored.